README.md

```diff
@@ -239,19 +239,78 @@ The Find All References and Go To Definition functionalities are extended to wor
 
 ## Processor Groups
 
-Use processor groups to link programs with specific dialects, SQL backend settings, and local folders containing copybooks. You define processor groups in a `proc_grps.json` file and associate them with programs in a `pgm_conf.json` file. Create both of these files in a `/.cobolplugin` folder in your workspace root.
+Use processor groups to link programs with specific dialects, SQL backend settings, copybook extensions, compiler options, and local folders containing copybooks. You define processor groups in a `proc_grps.json` file and associate processor groups with programs in a `pgm_conf.json` file. Create both of these files in a `/.cobolplugin` folder in your workspace root.
 
-In the `proc_grps.json` file: 
+The `proc_grps.json` file is formatted as an array of JSON elements, with one JSON per processor group. Each processor group can contain the following elements:
 
-- Specify your copybook folders in a `libs` array.
-- Specify dialects within a `preprocessor` JSON element. This element can contain a `libs` array which specifies folders that contain copybooks which use that dialect. 
-- Specify SQL backend settings within a separate `preprocessor` JSON element with the following parameters: 
-   - `name` (string)  
-   Specify **SQL** 
-   - `target-sql-backend` (string)  
-   Specify either **DB2_SERVER** or **DATACOM_SERVER**  
+- **"name":** (string)  
+    - Specify a name for the processor group.
+- (Optional) **"libs":** (array)  
+    - Specify libraries that contain copybooks as either absolute or relative local paths. These libraries are used to search for copybooks in programs linked with this processor group, and take priority over the local copybook libraries that you specify in the extension settings.
+- (Optional) **"copybook-extensions":** (array)  
+    - Specify copybook extensions that you use for the programs linked with this processor groups. These copybook extensions take priority over extensions that you specify in the extension settings.
+- (Optional) **"compiler-options":** (array)  
+    - Specify compiler directives that you want to apply to the programs linked with this processor group. Currently the following directives are supported:
```

Comment:
do we want to specify the default values for compiler-options?

Reply:
the default values according to the IBM doc are QUALIFY(COMPAT) and XMLPARSE(XMLSS). What I'm wondering is if the default values would ever need to be specified at all, and whether we can just say that you can add QUALIFY(EXTEND) and XMLPARSE(COMPAT) to this file to override them.

Is there a use case for explicitly enabling the default values? Like is it possible for a user to customize them differently on the backend?

Reply:
ok, as this is an optional item, it may be clear we are using IBM defaults...